autorag_research/pipelines/retrieval/bm25.py

```diff
@@ -0,0 +1,126 @@
+"""BM25 Retrieval Pipeline for AutoRAG-Research.
+
+This pipeline uses RetrievalPipelineService with BM25Module for retrieval.
+"""
+
+from typing import Any
+
+from sqlalchemy.orm import Session, sessionmaker
+
+from autorag_research.orm.service.retrieval_pipeline import RetrievalPipelineService
+
+
+class BM25RetrievalPipeline:
```

Comment:
I think we need parent class of the RetrievalPipeline to streamline each Retrieval Pipelines.